Makefile

```diff
@@ -133,6 +133,10 @@ ifndef LLAMA_NO_K_QUANTS
 	OBJS     += k_quants.o
 endif
 
+ifdef LLAMA_ROPE_SCALE
+	CXXFLAGS  += -DLLAMA_ROPE_SCALE=$(LLAMA_ROPE_SCALE)
```

Comment:
It should really be a command line option, IMHO.

Reply:
I agree, but like I mentioned I don't really think the patch would get merged in its current form. So using the define is mainly just to play with it.

Assuming it was decided this is something that should exist, then it would make sense to add more extensive changes to the examples. Right now it seems like exploring the idea is in the very preliminary stages.

Of course, if there was interest in merging this PR but making it configurable via commandline options was requested I'd absolutely be happy to make that change.

Reply:
If it's experimental, it doesn't matter.